src/Config/RuntimeConfigLoader.cs

```diff
@@ -42,18 +43,24 @@ public RuntimeConfigLoader(string? connectionString = null)
     public abstract string GetPublishedDraftSchemaLink();
 
     /// <summary>
-    /// Parses a JSON string into a <c>RuntimeConfig</c> object
+    /// Parses a JSON string into a <c>RuntimeConfig</c> object for single database scenario.
     /// </summary>
     /// <param name="json">JSON that represents the config file.</param>
     /// <param name="config">The parsed config, or null if it parsed unsuccessfully.</param>
     /// <returns>True if the config was parsed, otherwise false.</returns>
     /// <param name="replaceEnvVar">Whether to replace environment variable with its
```

Comment:
nit: the order of params in the summary here doesnt align with the actual order below

Reply:
fixed.